Cargo.toml

```diff
@@ -20,7 +20,7 @@ date_picker = ["chrono", "lazy_static", "icon_text"]
 color_picker = ["icon_text", "iced_graphics/canvas"]
 floating_button = ["button"]
 grid = []
-glow = [] # TODO
+glow = []                                                     # TODO
```

Comment:
What happened to this line? :smiley:

Reply:
huh idk that is weird i don't think the formatter i use by default worked correctly there i guess.